src/components/cld-analytics/cld-analytics.js

```diff
@@ -0,0 +1,82 @@
+// eslint-disable
+import { v4 as uuidv4 } from 'uuid';
+import Cookies from 'js-cookie';
+import { sendBeaconRequest } from './send-beacon-request';
+
+const CLD_ANALYTICS_ENDPOINT_URL = 'https://video-analytics-api.cloudinary.com/video-analytics';
+const CLD_ANALYTICS_USER_ID_COOKIE_KEY = 'cld-analytics-user-id';
+
+const getUniqueUserId = () => {
+  const cookieUserId = Cookies.get(CLD_ANALYTICS_USER_ID_COOKIE_KEY);
+
+  if (cookieUserId) {
+    return cookieUserId;
+  }
+
+  const userId = uuidv4();
+  const expirationDate = new Date(new Date().setFullYear(new Date().getFullYear() + 1));
+  Cookies.set(CLD_ANALYTICS_USER_ID_COOKIE_KEY, userId, { expires: expirationDate });
+  return userId;
+};
+
+const aggregateEvents = (eventsList) => {
+  const events = eventsList.reduce((acc, event) => {
+    const lastItem = acc.watchedFrames[acc.watchedFrames.length - 1];
+    if (event.type === 'PLAY') {
```

Comment:
Please take event names from 'PLAYER_EVENT' const

Reply:
as said above - this component is temporary put there - whenever we would like to share it for usage with other video players - we can easily extract it from there, thats the main reason for not using anything "internal" from cld-video-player